commit a26fd89c84bd3f8dfcc54d8aab84783db31c7c78
Author: ccobine <carol.cobine@joyent.com>
Date:   2019-03-06T14:54:56-05:00 (7 months ago)
    
    DOC-648 Please add that MD5 headers require base64 when adding a header for upload

diff --git a/docs/user-guide/storage-reference.md b/docs/user-guide/storage-reference.md
index 471b162..772bc72 100644
--- a/docs/user-guide/storage-reference.md
+++ b/docs/user-guide/storage-reference.md
@@ -83,7 +83,7 @@ which will be stored and returned back (HTTP content-negotiation will be handled
 If you do not specify a content type, the default is `application/octet-stream`.
 
 If you specify a `Content-MD5` header, the system validates that the content
-uploaded matches the value of the header.
+uploaded matches the value of the header. You must encode MD5 headers in Base64.
 
 
 The `durability-level` header is a value from 1 to 6
